Formula/woboq_codebrowser.rb

```diff
@@ -13,13 +13,18 @@ class WoboqCodebrowser < Formula
     sha256 "c04a6cd9fd7102e2ca5858dce5068bbfcede09e3d655e0512a2942d83f5a75e8" => :el_capitan
   end
 
-  depends_on "llvm"
   depends_on "cmake" => :build
+  depends_on "llvm"
 
   def install
-    system "cmake", ".", "-DLLVM_CONFIG_EXECUTABLE=#{Formula["llvm"].opt_bin}/llvm-config", "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON", *std_cmake_args
+    args = std_cmake_args + %W[
+      -DLLVM_CONFIG_EXECUTABLE=#{Formula["llvm"].opt_bin}/llvm-config
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+    ]
+    system "cmake", ".", *args
```

Comment:
Will `cmake` still do the same thing when `std_cmake_args` comes before the `-D`… constants?

Reply:
Aye, we use this syntax plenty in core without issue 👍.